scripts/prepareHoloFuelData/transact-holofuel.js

```diff
@@ -20,7 +20,7 @@ const transactHoloFuel = (agentTransactionLedger, type, { index, transactionTrac
     currentAgent = 1
   }
 
-  const txType = type === OFFER ? agentTransactionLedger[`offers`][`initated`] : agentTransactionLedger[type]
+  const txType = type === OFFER ? agentTransactionLedger.offers.initated : agentTransactionLedger[type]
```

Comment:
👋  hi @JettTech 
forgive me for saying something here, I know you're busy. I have been keeping up with various Holo and Holochain PRs, and watching along, and I spotted one thing here that seemed wrong not to point out, although it's possible it's already known. 

In this line it says `.offers.initated` and based on how it was before, that is correct in terms of the data, but `initated` should `initiated` if the word were spelled right. Anyways this seemed like the kind of thing that could easily cause a bug now or somewhere down the line, so it might be worth fixing in the upstream dependency. 
🤷‍♂ 
carry on, and feel free to ignore if this doesn't make sense.

Reply:
Good eye @Connoropolous! :eye:   ...and no, it has not been surfaced yet, so thanks for pointing that out!  Sometimes, I wish I could have a grammerly plugin for my code editor sometimes to manage typos! :joy_cat:  Nonetheless, great to know your scrupulous eyes are around... they always provide great support! :raised_hands: